<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseType</class>
 <widget class="QWidget" name="ChooseType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>83</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>83</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>83</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Choose Device Type</string>
  </property>
  <widget class="QPushButton" name="osciloscopeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Osciloscope</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multimeterButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Multimeter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signalGeneratorButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Signal Generator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
